#!/usr/env/python
PACKAGE = 'my_dyn_rec'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#List of parameters
gen.add('p_gain_accel', double_t, 0, "Accel P", 0.1, 0.0, 10.0)
gen.add('i_gain_accel', double_t, 0, "Accel I", 0.1, 0.0, 10.0)
gen.add('d_gain_accle', double_t, 0, "Accel D", 0.1, 0.0, 10.0)
gen.add('p_gain_brake', double_t, 0, "Brake P", 0.1, 0.0, 10.0)
gen.add('i_gain_brake', double_t, 0, "Brake I", 0.1, 0.0, 10.0)
gen.add('d_gain_brake', double_t, 0, "Breke D", 0.1, 0.0, 10.0)



exit(gen.generate(PACKAGE, "my_dyn_rec", "MyParams"))